stanford-earth/SE3_BLT
# Enabled and exported stanford_earth_views_content

# READY FOR REVIEW

# Summary
- New view page content type.

# Needed By (Date)
- End of sprint

# Urgency
- Mild

# Steps to Test

1. Checkout this branch and same branch for [matson](https://github.com/stanford-earth/matson/pull/91) & [stanford_earth_helper](https://github.com/stanford-earth/stanford_earth_helper/pull/4)
2. drush cim or drush en stanford_earth_views_content
3. create new content type of view page
4. choose an appropriate view and add some top media/text etc
5. verify page looks good.

# Associated Issues and/or People
- https://github.com/stanford-earth/stanford_earth_helper/pull/4
- https://github.com/stanford-earth/matson/pull/91

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
